python/activity_stream/ui/note_widget.py

```diff
@@ -62,6 +62,18 @@ def setupUi(self, NoteWidget):
         self.date.setObjectName("date")
         self.horizontalLayout.addWidget(self.date)
         self.verticalLayout.addLayout(self.horizontalLayout)
+        self.content_editable = QtGui.QPlainTextEdit(self.frame)
```

Comment:
I am curious about the design and tech decisions here:

We have a standardized [note widget](http://developer.shotgunsoftware.com/tk-framework-qtwidgets/note_input_widget.html) that supports screenshots and auto completion:

![image](https://cloud.githubusercontent.com/assets/337710/18746573/17157562-80c0-11e6-9d00-4101735b2e21.png)

What's the reason we are not calling up our standard note editor? @thebeeland @robblau ?

As a user, i would like UX consistency. This would lead to simpler code. If i update a note, create one or reply to one, i'd expect to see the same UI and have access to the same functionality. Like with note replies, it didn't even have to embed into the UI itself *(further down the code there are issues with font sizing magic which is unlikely to work across DCCs)*. It seems much simpler if we would just add the little update button to the note widget but then we just call out to the note widget to do the job in a modal window.

Implementation would be very straight forward, just use this:
- https://github.com/shotgunsoftware/tk-framework-qtwidgets/blob/master/python/activity_stream/dialog_reply.py#L14

- https://github.com/shotgunsoftware/tk-framework-qtwidgets/blob/master/python/activity_stream/reply_list.py#L287

- https://github.com/shotgunsoftware/tk-framework-qtwidgets/blob/master/python/activity_stream/reply_list.py#L395

here's where that code runs: 

![image](https://cloud.githubusercontent.com/assets/337710/18746899/f47cb1f8-80c1-11e6-8870-d09ad2926c33.png)

In addition to what is already there, I guess we would need to extend the note editor widget to take a pre-loaded text as it pops up.

I understand there is a desire to get this out of the door quickly, but I think we need to disuss the right tech strategy for this. At the very least we need to get some work scheduled to improve the current implementation I think. Curious to hear what yall think! :)